sdk/tests/deqp/functional/gles3/es3fNegativeShaderApiTests.js

```diff
@@ -1119,9 +1120,10 @@ goog.scope(function() {
                 gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, tfID);
                 this.expectError(gl.NO_ERROR);
 
-                bufferedLogToConsole('gl.INVALID_VALUE is generated if program is not the name of a program object.');
-                name = gl.getTransformFeedbackVarying(null, 0);
-                this.expectError(gl.INVALID_VALUE);
+                bufferedLogToConsole('An exception is thrown if program is null.');
+                this.expectThrowNoError(function() {
+                    name = gl.getTransformFeedbackVarying(null, 0);
```

Comment:
I don't think an assignment to the variable "name" here works. Since "name" isn't used afterward, could you please just remove the assignment?